manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -644,6 +644,17 @@ Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
         });
     }
 
+    ,rendLink: function(v,attr) {
```

Comment:
A better name for this function would be `renderLink` (or is there a particular reason why it's called `rendLink`?).
```suggestion
    ,renderLink: function(v,attr) {
```

Reply:
Yes, probably the new name is clearer, I will correct it later.

Reply:
It's probably called that because that's the pattern used for other renderers too (e.g. `rendYesNo`)